src/CftfBundle/Controller/DocTreeController.php

```diff
@@ -210,7 +210,7 @@ public function exportAction(Request $request, LsDoc $lsDoc)
             'itemTypes' => $itemTypes,
             'subjects' => $lsDoc->getSubjects(),
             'concepts' => [],
-            'licences' => [],
+            'licence' => $lsDoc->getLicence(),
```

Comment:
Per the spec, it is possible for each item to have a different licence than the document, hence the array that was here.